src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/logging/CheckLoggingSpec.groovy

```diff
@@ -86,4 +88,26 @@ class CheckLoggingSpec extends HealthCheckSpecification {
                     "Storm should generate a warning message about not being able to find the flow"
         }
     }
+
+    def "Check push discovery package logging"() {
+        given: "Two port(with/without ISL)"
+        def sw = topology.activeTraffGens*.switchConnected.first()
+        def portWithIsl = topology.getRelatedIsls(sw).first().srcPort
+        def portWithoutIsl = 10 //default port for traffgen
```

Comment:
Default traffgen port may change, it is configurable. Instead of doing such an assumption you should get switch ports and find a port without isl